bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/general/GeneralRule.java

```diff
@@ -547,9 +547,9 @@ public IP2Problems check(final ObservationElements elements)  {
             final ISPTemplateFolder templateFolderNode = elements.getProgramNode().getTemplateFolder();
             if (templateFolderNode != null) {
 
-                final SPTarget obsTarget = getObsTarget(elements);
-                if (obsTarget != null) {
-
+                // TODO:GHOST Do we have to do anything for SPCoordinates here as well?
+                final Asterism a = getObsAsterism(elements);
+                a.allSpTargetsJava().foreach(obsTarget -> {
```

Comment:
I'm guessing the answer to your question is no. This rule is about not slipping in new targets that weren't approved and that should be accomplished with the real science targets.

It does bring up the problem of matching this to the P1 model. We don't have the asterism concept in P1 yet so we really can't do this check. It would require two TemplateParameters for each dual target, each with its own conditions and observation time. Since it doesn't make sense maybe we should skip this check for now for anything but single target asterism.